src/irmin/node_intf.ml

```diff
@@ -177,6 +179,18 @@ module type Portable = sig
     node:node_key option Merge.t ->
     t Merge.t
   (** [merge] is the merge function for nodes. *)
+
+  (** {1 Proofs} *)
+
+  type proof =
+    [ `Blinded of hash
+    | `Values of (step * value) list
```

Comment:
shouldn't `value` here be redefine to only contain hashes (instead of keys) as in [inodes](https://github.com/mirage/irmin/pull/1716/commits/a0d4006d1629c90daeb35feac77a79aff5891224?show-viewed-files=true&file-filters%5B%5D=#diff-032a7a23087d2e5ce953b0543a6183b1b5b12a66610f55dfef414ddb2d2a707cR1524)?

Reply:
It is already the case since `Core` is included with the substitution of `contents_key` and `node_key` to `hash`

Reply:
Thanks!